toolchain/driver/driver.cpp

```diff
@@ -131,10 +133,11 @@ auto Driver::RunDumpSubcommand(DiagnosticConsumer& consumer,
                        .Case("parse-tree", DumpMode::ParseTree)
                        .Case("semantics-ir", DumpMode::SemanticsIR)
                        .Case("llvm-ir", DumpMode::LLVMIR)
+                       .Case("assembly", DumpMode::Assembly)
                        .Default(DumpMode::Unknown);
   if (dump_mode == DumpMode::Unknown) {
     error_stream_ << "ERROR: Dump mode should be one of tokens, parse-tree, "
-                     "semantics-ir, or llvm-ir.\n";
+                     "semantics-ir, llvm-ir or assembly.\n";
```

Comment:
```suggestion
                     "semantics-ir, llvm-ir, or assembly.\n";
```

Missed this one before -- we prefer Oxford commas.